src/NuGet.Clients/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs

```diff
@@ -597,7 +597,7 @@ internal static IEnumerable<EnvDTEProject> GetSupportedChildProjects(EnvDTEProje
 
         internal static MicrosoftBuildEvaluationProject AsMicrosoftBuildEvaluationProject(string dteProjectFullName)
         {
-            // Need NOT be on the UI thread
+            // Should be called from the UI thread
```

Comment:
@ rrelyea yes, there is only one other caller, and it calls this from a UI thread within a JTF.